utils/flags/flags.go

```diff
@@ -189,6 +189,18 @@ func GetCliFlags() []cli.Flag {
 			Usage:   "The base URL to use for a http proxy backend.",
 			EnvVars: []string{"BAZEL_REMOTE_HTTP_PROXY_URL"},
 		},
+		&cli.StringFlag{
+			Name:    "http_proxy.key_file",
+			Value:   "",
+			Usage:   "Path to the key used to autheticate with the proxy backend. Enables mTLS.",
```

Comment:
The help text for these two new options should each mention that the other must also be specified.